source/getting-started/concepts-terminology.markdown

```diff
@@ -64,6 +68,10 @@ Similar to {% term automations %}, scripts are repeatable {% term actions %} tha
 
 ![Scripts](/images/getting-started/script_01.png)
 
+## Actions
```

Comment:
This term is already used in the automations concept above, seems his is introduced to late in that case?

I'm not sure if this needs its own definition, as it is currently introduced as part of automation. To put this into perspective, we don't write about triggers or conditions separately either...

Reply:
That's a fair point. Isn't there also a dedicated page for actions? Maybe we could instead just link there.